libcodechecker/libhandlers/server.py

```diff
@@ -247,6 +256,30 @@ def add_arguments_to_parser(parser):
                                 help="Stops all of your running CodeChecker "
                                      "server instances.")
 
+    database_mgmnt = parser.add_argument_group(
+            "Database management arguments.",
+            """WARNING these commands needs to be called with the same
+            workspace and configuration arguments as the server
```

Comment:
workspace has been in the process of being removed, I'd rather not mention it in a new help thing.